src/pilot/assign_role.py

```diff
@@ -428,16 +457,17 @@ def define_storage_logical_disks(drac_client, raid_controller_name):
     #
     # A successful call returns a list, which may be empty; otherwise,
     # None is returned.
-    jbod_capable = drac_client.is_jbod_capable(raid_controller_name)
-    jbod_logical_disks = define_jbod_logical_disks(
-        drac_client, remaining_physical_disks, raid_controller_name,
-        jbod_capable)
+    logical_disks = [os_logical_disk]
+    if raid_controller:
```

Comment:
this would change to looping through the raid controllers

Reply:
Done